WebContent/js/containers/DatasetTree.js

```diff
@@ -117,6 +117,7 @@ class DatasetTree extends React.Component {
                         <div className="component-header">
                             <UpperCaseTextField
                                 id="datasets-qualifier-field"
+                                style={{ width: '100%', paddingLeft: '8px' }}
```

Comment:
style={{ flexGrow: 1, paddingLeft: '8px' }} 
producing same result

https://stackoverflow.com/questions/34733955/what-are-the-differences-between-flex-grow-and-width
```
> Yes, if there is one element in the row, width: 100% and flex-grow: 1 may have the same effect (depending on padding, border and box-sizing settings).

But what if there are two elements, and you want the second one to take the remaining space? With a sibling in the container, width: 100% causes an overflow. I guess you can do something like this:
width: calc(100% - width of sibling);
```
Though suggested if more than one element flex-grow is preferred.
but unable to cause any overflow issues with width 100%